classes/cliTool/traits/ConvertLogFile.php

```diff
@@ -69,12 +69,6 @@ abstract public function getParseRegex(): string;
      */
     abstract public function getPhpDateTimeFormat(): string;
 
-    /**
-     * Weather the URL parameters are used instead of CGI PATH_INFO.
-     * This will determine how URLs are parsed.
-     */
-    abstract public function isPathInfoDisabled(): bool;
-
```

Comment:
Nothing should be removed from this file -- I extra added the functions here to be able to parse the old log files that used disabled path info once all the code is removed :-)